mozilla/fx-private-relay
# Improve scrolling experience across browsers

This PR fixes #3407, fixes #3406, fixes #3223, fixes #3079.

Updates layout structure and styling to avoid the issues above. It also moves the menu for small/mobile screens inside the header so that it is able to scroll smoothly with the header.

How to test:

1. Go to Private Relay website (on iOS / Windows / Android browser).
2. Scroll through each page (including tracker report)
3. Observe the following:
   1. Short pages have the footer reach the bottom of the screen
   2. Mobile menu remains directly under the header during scrolling
   3. There are not multiple scroll bars on the page
   4. No blank areas while scrolling on mobile

- [ ] ~l10n changes have been submitted to the l10n repository, if any.~ (N/A)
- [ ] ~I've added or updated relevant docs in the docs/ directory.~ (N/A)
